src/command-line/add.js

```diff
@@ -1,14 +1,21 @@
 "use strict";
 
-var ClientManager = new require("../clientManager");
-var colors = require("colors/safe");
-var program = require("commander");
-var Helper = require("../helper");
+const colors = require("colors/safe");
+const program = require("commander");
+const fs = require("fs");
+const Helper = require("../helper");
 
 program
 	.command("add <name>")
 	.description("Add a new user")
 	.action(function(name) {
+		if (!fs.existsSync(Helper.USERS_PATH)) {
+			log.error(`${Helper.USERS_PATH} does not exist.`);
+			return;
+		}
```

Comment:
We mentioned making the default mode private, which I agree, however this PR would make the setup workflow even more convoluted:

0. `lounge start` fails because there is no user (we should still start the instance even without user set up, btw, since we always user files #779)
1. Update configuration file to temporarily setup The Lounge in public
2. `lounge start` to create config _and_ start a public instance
3. Kill it
4. Update configuration file to set to private mode
5. `lounge add <user>`
6. `lounge start` to start private mode with that user